api/base/views.py

```diff
@@ -505,7 +505,7 @@ def get_object(self):
 
 class BaseContributorList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin):
 
-    ordering = ('-modified',)
+    ordering = ('-node__modified',)
```

Comment:
user__modified makes more sense, since this is listing contributors for a node, which will then all have the same `modified` value

Reply:
ping